bin/openstack-install

```diff
@@ -61,6 +61,9 @@ def parse_options(argv):
                         help="Custom configuration for OpenStack installer.")
     parser.add_argument('--charm-config', type=str, dest='charm_config_file',
                         help="Additional charm settings")
+    parser.add_argument('--charm-plugin-dir', type=str, dest='charm_plugins',
```

Comment:
Can we rename 'charm_plugins' to maybe 'charm_plugin_dir', since it's a little unclear when you're reading the other parts of the code?